### PR TITLE
Added checklist to CONTRINUTING.md

### LINKED ISSUE #9660: Adding how to mark checkboxes in contributing.md
### Feature description

To add checkboxes in a GitHub Markdown file like the one you provided, you can use the following syntax:
- [ ] Task 1
- [x] Task 2 (completed)
- [ ] Task 3
In the Markdown syntax:

[ ] represents an empty checkbox, indicating an incomplete task.
[x] represents a checked checkbox, indicating a completed task.

### PR DESCRIPTION
**Pull request by Asif-Sheriff**
### Describe your change:

Fixes #9660

Added checklist to CONTRINUTING.md

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [X] Documentation change?

### Checklist:
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Contributing guidelines
 
+## Contribution Checklist
+
+- [ ] Read the [Code of Conduct](CODE_OF_CONDUCT.md).
+- [ ] Fork the repository.
+- [ ] Clone the repository to your local machine.
```
Comment (cclauss) [inline CONTRIBUTING.md:7]: * `pipx install pre-commit`
* `pre-commit install`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Contributing guidelines
 
+## Contribution Checklist
+
+- [ ] Read the [Code of Conduct](CODE_OF_CONDUCT.md).
```
Comment (tianyizheng02) [inline CONTRIBUTING.md:5]: Where does this link point to? We don't have a code of conduct file.

**Review by tianyizheng02 — COMMENTED** [review]
I don't think this actually address the issue. I understood the original issue to mean that we should add a section to the contributing guidelines about _how_ to fill out the checkboxes in the PR template. I don't think it was asking for a step-by-step guide on how to contribute.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Asif-Sheriff) [reply CONTRIBUTING.md:5]: on second thoughts I believe you are correct
I will make the necessary changes

**Comment by tianyizheng02** [comment]
Closing this PR because of lack of response to code review and because it doesn't seem to properly address the original issue

**Pull request closed without merging**